src/hooks/http.js

```diff
@@ -267,12 +292,20 @@ export const httpRequestWrapper = originalRequestFn =>
         logger.warn('end wrap error', e.message);
       }
 
+      if (!isRequestToAwsService) {
+        try {
+          const emitWrapper = httpRequestEmitWrapper(requestData);
+          shimmer.wrap(clientRequest, 'emit', emitWrapper);
```

Comment:
you're adding another shimmering - can you stress test that?

Reply:
Yes, I'm adding in integration-test